### ISSUE TITLE
Cannot find Scrt1.o & crti.o

### ISSUE THREAD

**Issue by jrang188** [JanDeDobbeleer/homebrew-oh-my-posh#24]
Hi everyone, I got this error when I was installing oh-my-posh in Fedora using LinuxBrew.
![image](https://github.com/JanDeDobbeleer/homebrew-oh-my-posh/assets/59113087/018c5730-8510-44b6-af9a-d0bd71b66d17)

**Comment by jrang188** [comment]
nvm, it works now

**Comment by mazansm** [comment]
I have this same issue... how were you able to resolve?

**Comment by JanDeDobbeleer** [comment]
@mazansm you need to have a closer look at the additional instructions on how to correctly set up homebrew on linix

**Comment by mazansm** [comment]
Thanks so much for getting back to me. Huge fan of your work and OMP! Anyway, I am not unfamiliar with getting homebrew and OMP working successfully - I have it running successfully on Ubuntu for example, but I am having an issue with openSUSE Tumbleweed. I went back and verified the dependencies based on your response and feel like I have everything, but I added 2 or 3 additional items just to be sure. Here is a screenshot of my error. Also, if it helps, I was able to successfully install brew hello and receive the Hello World response.

Here is the output from this command:

brew install jandedobbeleer/oh-my-posh/oh-my-posh

![image](https://github.com/JanDeDobbeleer/homebrew-oh-my-posh/assets/27631264/2a574db6-5d7e-4c55-9fb5-0774e15699e7)
